prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -400,7 +399,8 @@ void addNewPendingEntityInstances(Date to, String entityType) throws FalconExcep
                                 currentCluster.getName()).getEnd();
                     }
                     while (nextInstanceTime.before(to) && nextInstanceTime.before(endDate)) {
-                        LOG.info("Adding instance={} for <entity,cluster>={}", nextInstanceTime, key);
+                        LOG.debug("Adding pending instance={} for <entity,cluster>={}; entityType={}",
+                                nextInstanceTime, key, entityType);
```

Comment:
I believe we should change this one to trace as this will print so many lines in the logs.